src/main/resources/mixins.vanilla.chunkio.json

```diff
@@ -0,0 +1,17 @@
+{
+    "required": true,
+    "minVersion": "0.5.1",
+    "package": "org.spongepowered.server.mixin.chunkio",
+    "refmap": "mixins.vanilla.refmap.json",
+    "priority": 1112,
+    "server": [
+        "MixinAnvilChunkLoader_ChunkIO",
```

Comment:
Is the `_ChunkIO` suffix really needed? The mixins are in a `chunkio` package already.